test/integration/gpu_lidar_sensor_plugin.cc

```diff
@@ -538,7 +538,9 @@ void GpuLidarSensorTest::TestThreeBoxes(const std::string &_renderEngine)
   // Sensor 1 should see box01 and box02
   EXPECT_NEAR(sensor1->Range(0), expectedRangeAtMidPointBox2, LASER_TOL);
   EXPECT_NEAR(sensor1->Range(mid), expectedRangeAtMidPointBox1, LASER_TOL);
+#ifndef __APPLE__
```

Comment:
do you think it's worth adding links to #66 / #67 with these ifdefs?

Reply:
I thought of that too, let me add it